app/build.gradle

```diff
@@ -242,19 +242,16 @@ android.applicationVariants.all { variant ->
         // The Google Play Store does not allow multiple APKs for the same app that all have the
         // same version code. Therefore we need to have different version codes for our ARM and x86
         // builds.
-        // Our generated version code has a length of 8 (See tools/gradle/versionCode.gradle).
-        // We will prefix our ARM builds with 1 and our x86 builds with 2. Our x86 builds need a
-        // higher version code to avoid installing ARM builds on an x86 device with ARM
-        // compatibility mode.
-        def multiplier = 100000000
+
+        // Our generated version code now has a length of 9 (See tools/gradle/versionCode.gradle).
+        // Our x86 builds need a higher version code to avoid installing ARM builds on an x86 device
+        // with ARM compatibility mode.
 
         if (variant.flavorName.contains("X86")) {
-            versionCode = versionCode + (3 * multiplier)
+            versionCode = versionCode + 2
         } else if (variant.flavorName.contains("Aarch64")) {
-            versionCode = versionCode + (2 * multiplier)
-        } else if (variant.flavorName.contains("Arm")) {
-            versionCode = versionCode + (1 * multiplier)
-        }
+            versionCode = versionCode + 1
+        } else // variant.flavorName.contains("Arm")) use generated version code
```

Comment:
This lonely `else` .. is that valid Groovy code?

Reply:
Oops, I'll fix that.